main.go

```diff
@@ -19,9 +21,19 @@ var conf = config.Get()
 func main() {
 	logging.InitLogger(conf)
 
-	dao.Init()
+	setUpDatabase := flag.Bool("setup", false, "create the schema and run the migrations")
+	resetDatabase := flag.Bool("reset", false, "drop the schema, recreate it and run the migrations")
+
+	flag.Parse()
```

Comment:
Final comment: we can move these over to the `config/` package here: https://github.com/RedHatInsights/sources-api-go/blob/62ea4f6533db5dbddb5d5dba4682f6045eca432b/config/config.go#L111

That way they only get parsed once and the config struct has them (like you do below).

Reply:
Done!